### PR TITLE
Regular updates (May 25)

### PR DESCRIPTION
**Pull request by jorgeorpinel**
- Fix DVC repo tooltip
- Correct run --no-exec option behavior
- Misc. fixes

### CODE REVIEW

File: content/docs/command-reference/run.md [hunk 1]
```diff
@@ -150,12 +150,12 @@ data pipeline (e.g. random numbers, time functions, hardware dependency, etc.)
...
-  stage is actually executed. DVC will also add your outputs to `.gitignore`,
-  same as it would do without `--no-exec`. This is useful if, for example, you
-  need to build a pipeline (dependency graph) first, and then run it all at
-  once.
+  it, nor commit dependencies or outputs to DVC. In the DVC-file contents, the
+  file hash values will be empty (like with `--no-commit`, explained below);
+  They will be populated the next time this stage is actually executed or with
+  `dvc commit`. DVC will also add your outputs to `.gitignore`, same as it would
+  do without `--no-exec`. This is useful if, for example, you need to build a
+  pipeline (dependency graph) first, and then run it all at once.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/run.md:158]: This is per https://discord.com/channels/485586884165107732/485596304961962003/715285386053550140

Is it correct?

Comment (shcheklein) [reply content/docs/command-reference/run.md:158]: This is good!

`commit dependencies or outputs` - we don't use `commit` term in this sense anywhere else - better to say "save" or "cache" ... or be more explicit if we don't have a good term

Comment (jorgeorpinel) [reply content/docs/command-reference/run.md]: Updated in 980c1fb9

File: content/authors/dmitry_petrov.md [hunk 2]
```diff
@@ -5,5 +5,5 @@ link: https://twitter.com/fullstackml
 ---
 
 Creator of [http://dvc.org](http://dvc.org) — Git for ML. Ex-Data Scientist
-[http://twitter.com/Microsoft](@Microsoft). PhD in CS. Making jokes with a
+[@Microsoft](http://twitter.com/Microsoft). PhD in CS. Making jokes with a
 serious face.
```
Comment (jorgeorpinel) [inline content/authors/dmitry_petrov.md:9]: Probably what was meant 😬

**Pull request merged**